types/src/cryptography/hash.rs

```diff
@@ -25,6 +26,27 @@ impl Hashable for Address {
     }
 }
 
+impl Hashable for u128 {
+    fn hash_custom(&self) -> Hash {
+        hash(self.to_be_bytes())
```

Comment:
Out of curiosity, does the endianess matter here? Using little-endian could be slightly faster on most CPUs (most are LE).

Reply:
I'm not sure how much, but I chose BE to follow a convention: https://crypto.stackexchange.com/questions/2099/why-do-all-hash-functions-use-big-endian-data